From b8e32cb696b1b6fe0a450cab9662971ab2a2d5a6 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Mon, 30 Jan 2023 15:16:07 -0800
Subject: [PATCH] [GTK] Fix attempt to include WebKitPrintCustomWidget.h
 https://bugs.webkit.org/show_bug.cgi?id=251392

Unreviewed. This is a follow-up to 259286@main and 259551@main.

* Source/WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h:

Canonical link: https://commits.webkit.org/259590@main
---
 .../WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Source/WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h b/Source/WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h
index 3eafd07f9db0..4b82916bce9b 100644
--- a/Source/WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h
+++ b/Source/WebKit/UIProcess/API/gtk/WebKitPrintCustomWidgetPrivate.h
@@ -19,8 +19,12 @@
 
 #pragma once
 
+#if !ENABLE(2022_GLIB_API)
+
 #include "WebKitPrintCustomWidget.h"
 #include <gtk/gtk.h>
 
 void webkitPrintCustomWidgetEmitCustomWidgetApplySignal(WebKitPrintCustomWidget*);
 void webkitPrintCustomWidgetEmitUpdateCustomWidgetSignal(WebKitPrintCustomWidget*, GtkPageSetup*, GtkPrintSettings*);
+
+#endif
